packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -724,6 +726,25 @@ export default {
             this.trackFormInteraction({ action: 'error', error: `error_${error.message}` });
 
             this.genericErrorMessage = messageToDisplay || this.$t('errorMessages.genericServerError');
+
+            this.$nextTick(() => {
+                this.scrollToElement('errorAlert');
+            });
+        },
+
+        /**
+         * Scroll to a ref element in the screen.
+        */
+        scrollToElement (refElement) {
+            const scrollingDurationInMilliseconds = 650;
+
+            const element = this.$refs[refElement]
+                ? this.$refs[refElement].$el
+                : null;
+
+            if (element) {
+                VueScrollTo.scrollTo(element, scrollingDurationInMilliseconds, { offset: -20 });
```

Comment:
You could avoid importing `vue-scrollto` and use `this.$scrollTo` here.

Reply:
This didn't work for me. We have a bit of a weird way of including things in our components, in my opinion, given we don't really use `Vue.use(...)` anywhere. I had tried adding it to `index.js` but didn't do anything, so I had to import it directly into my component and use `VueScrollTo.xxxxxxxx`.

Any opinions? cc @tommcclean, @DamianMullins.